src/super_gradients/training/transforms/keypoints/keypoints_mixup.py

```diff
@@ -44,17 +46,26 @@ def __init__(self, prob: float):
 
         :param prob:            Probability to apply the transform.
         """
-        super().__init__(additional_samples_count=1)
+        super().__init__()
         self.prob = prob
 
+    @property
+    def additional_samples_count(self) -> int:
+        do_mixup = random.random() < self.prob
+        return int(do_mixup)
+
+    @additional_samples_count.setter
+    def additional_samples_count(self, value):
+        pass  # ignored
```

Comment:
Maybe let's throw an exception to explicitly disable writing to this property?

Reply:
Done, [49f5548](https://github.com/Deci-AI/super-gradients/pull/1897/commits/49f55484f022f5938ef472a5eed5900a26fe9d2b)